<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>1301</width>
     <height>471</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nama</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Harga</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Deskripsi</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="searchText">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allMenuButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>All Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHarga">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jumlah : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="jumlahText">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>120</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addToCartButton">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>120</y>
     <width>161</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Add To Cart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gtCartButton">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>180</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to cart</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNama">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
